spec/Encoding/MemoryStream.php

```diff
@@ -10,7 +10,9 @@ class MemoryStream implements StreamInterface
 
     private $size = 0;
 
-    public function __construct($body = "", $chunkSize = null)
+    private $chunkSize;
+
+    public function __construct($body = "", $chunkSize = 8192)
```

Comment:
how did you determine this default? is there a way to figure out the system default instead of hardcoding it here? i guess 8kb is the norm for 64bit and 4096 for 32bit?

Reply:
Default value is the `BUFSIZ` C constant which depends on the compiler used, and there is no way to get it here.

When reading various streaming PHP librairies, 8192 was a widely value used for buffer, so decided to keep here.

Reply:
thanks for the explanation!
oh this is the spec and not the source. sorry i asked, for a test we don't need to overthink it anyways.